### ISSUE TITLE
no documentation for `..{n} -> `  irrelevance syntax  -- or in the manual

### ISSUE THREAD

**Issue by cartazio** [agda/agda#2635]
eg in https://github.com/UlfNorell/agda-prelude/blob/master/src/Prelude/Fin.agda#L13

```agda
data Fin : Nat → Set where
  | zero : ∀ ..{n} → Fin (suc n)
  | suc  : ∀ ..{n} (i : Fin n) → Fin (suc n)
```

also  theres no docs on irrelevance and current syntax in the docs... just in the change logs :)

**Comment by andreasabel** [comment]
I think Ulf's use of the `..` modality isn't quite safe yet, see #2640.
And mine neither: #2595.

**Comment by UlfNorell** [comment]
For future reference, #2640 is a problem with `..` in general and not with my particular use.

**Comment by jespercockx** [comment]
This is a duplicate of #1114.

**Comment by andreasabel** [comment]
We should document `..` once we have resolved #2757 (and actually know what `..` is).

**Comment by jespercockx** [comment]
Well, $..$ is related to $.$ the same way $#$ is related to regular relevant functions. So in principle, once we know how parametricity works, explaining $..$ shouldn't be hard.
